### PR TITLE
feat(add-chain): adding vechain network to the chain list

### PR DESCRIPTION
**Pull request by darrenvechain**
## PR-Codex overview
This PR focuses on adding support for the Vechain blockchain. 

### Detailed summary
- Added `vechain` export to `src/chains/index.ts`
- Added `vechain.ts` file in `src/chains/definitions` directory
- Imported `defineChain` function from `../../utils/chain.js`
- Defined `vechain` chain object with properties such as id, name, network, nativeCurrency, rpcUrls, and blockExplorers

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a patch changeset with `pnpm changeset`. See the instructions here: https://github.com/wagmi-dev/viem/blob/main/.github/CONTRIBUTING.md#4-export-your-chain

File: src/chains/definitions/vechain.ts
Comment (tmm) [inline src/chains/definitions/vechain.ts:1]: Also need to run `pnpm lint:fix`. This file looks like it has formatting issues.

**Review by tmm — CHANGES_REQUESTED** [review]
Should run `pnpm test src/chains/index.test.ts`

**Comment by jxom** [comment]
Thanks for the PR! Can you please re-open this pull request with @tmm's requested changes? (Closing for now to prevent stale PRs)

**Pull request closed without merging**